fetch.sh

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 set -eux
 
+if [ ! "$(which bsdtar)" ] && [ ! "$(which unzip)" ]; then
```

Comment:
I think this acts kind of strangely, the && is making the second /usr/bin/[ conditional a separate statement. I'd suggest rewriting as:
`if [ -f "$(which bsdtar)" -a -f "$(which unzip)" ]; then`

Reply:
Cool, I can do that.